dashboard/db/migrate/20160718080224_remove_user_id_from_sections.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUserIdFromSections < ActiveRecord::Migration
+  def up
+    remove_column :sections, :user_id
+  end
+
+  def down
+    add_column :sections, :user_id, :integer
+  end
+end
```

Comment:
I think that's a good idea. I'll probably be combining these migrations into a larger one with an updated timestamp. Right now these two migrations aren't consecutive based on when they were created.